v22.2/choose-a-deployment-option.md

```diff
@@ -52,6 +52,28 @@ Cockroach Labs offers three ways to deploy CockroachDB: two managed services&mda
         <li><b>Advanced features</b>: Yes. See <a href="enterprise-licensing.html">Enterprise Features</a>.</li>
       </ul></td>
   </tr>
+  <tr>
+      <td><ul>
+        <li>All workloads: lightweight and critical production.</li>
+        <li>Applications that may need to grow and scale over time.</li>
+        <li>Applications with current and future requirements to grow into new cloud regions to serve customers in new markets.</li>
+        <li>Applications that require real-time integration with other systems.</li>
+        <li>Applications that need to be PCI compliant.</li>
+      </ul></td>
+      <td><ul>
+        <li><a id="dedicated-advanced"></a><b><a href="../cockroachcloud/quickstart-trial-cluster.html">{{ site.data.products.dedicated }} advanced</a></b>: A fully managed, single tenant CockroachDB deployment in a single region or multi-region cloud (AWS or GCP) with PCI ready features.</li>
+      </ul></td>
+      <td><ul>
+        <li><b>Scale</b>: Node-based; self-service add and remove nodes.</li>
+        <li><b>Availability</b>: Service availability guaranteed with 99.99% uptime. Configurable data replication within or across regions.</li>
+        <li><b>Operations</b>: Cockroach Labs SRE provides guaranteed uptime, optimization, security, and operations for cluster, node, and cloud instances. Backups daily and hourly.</li>
+        <li><b>Cost</b>: Pricing based on disk size and storage. A single, predictable price packages hardware costs with SRE resources and support.</li>
+        <li><b>Resource isolation</b>: Dedicated, single-tenant instance of CockroachDB software and infrastructure.</li>
+        <li><b>Support</b>: Enterprise grade <a href="https://support.cockroachlabs.com/">support</a> provided by Cockroach Labs.</li>
+        <li><b>Advanced security</b>: Access to all features required for PCI compliance, including CMEK and egress rules.</li>
```

Comment:
"PCI readiness" instead of "PCI compliance". "Egress Perimeter Controls" instead of "egress rules" for consistency.